ui/src/pages/publications/[id]/index.tsx

```diff
@@ -234,6 +243,31 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
         }
     }, [confirmation, publicationData?.id, router]);
 
+    const handleUnlock = async () => {
+        const confirmed = await confirmation(
+            'Unlocking this publication for editing will invalidate all existing author approvals. Authors will be invited to approve your new changes once you have finished editing',
+            'Are you sure you want to cancel and unlock?',
+            <OutlineIcons.CloudUploadIcon className="h-10 w-10 text-grey-600" aria-hidden="true" />,
```

Comment:
I don't think 'CloudUploadIcon' matches the confirmation modal. This needs to be checked with PO